definitions/variable/emissions/carbon-management.yaml

```diff
@@ -1,5 +1,3 @@
-# Carbon Capture is grouped into two levels of aggregation based on the fate of the captured CO2
-# storage versus utilization
 - Carbon Capture:
     description: Total amount of captured carbon dioxide (CO2) from energy use in supply
```

Comment:
Suggestion: Total amount of carbon dioxide (CO2) captured either from the atmosphere, directly from oceans, or from point sources as energy use in supply and demand sectors (IPCC category 1A, 1B) or industrial processes

Reply:
Done.